sdk/executor/src/test/java/com/mesosphere/sdk/executor/CustomExecutorTest.java

```diff
@@ -48,7 +48,7 @@ public void testSimple() {
                         .setEnvironment(Protos.Environment
                                 .newBuilder()
                                 .addVariables(
-                                        TaskTestUtils.createEnvironmentVariable(DcosTaskConstants.TASK_TYPE, "TEST")))
+                                        TaskTestUtils.createEnvironmentVariable("TASK_TYPE", "TEST")))
```

Comment:
One benefit to leaving this as a constant, even though it leads to redundant typing, is that it can be changed in one place only if it needs to be changed.